crates/db_schema/src/impls/local_user.rs

```diff
@@ -62,6 +63,15 @@ impl LocalUser {
       .get_result(conn)
       .await
   }
+
+  pub async fn get_by_person_id(pool: &mut DbPool<'_>, id: &PersonId) -> Result<Self, Error> {
```

Comment:
You can remove this, and just use `LocalUserView::read_person`

Reply:
Fixed